### PR TITLE
Remove loop argument

### LINKED ISSUE #38: Remove loop argument
To remove warning at latest Python version and to be ready for the future Python versions: `DeprecationWarning: The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.`

### PR DESCRIPTION
**Pull request by decaz**
... and some other fixes.

Closes #38.

### CODE REVIEW

File: aiozk/recipes/base_watcher.py [hunk 1]
```diff
@@ -10,25 +10,25 @@
 log = logging.getLogger(__name__)
 
 
-def maybe_future(fut, loop):
+def maybe_future(fut):
     if inspect.isawaitable(fut):
-        asyncio.ensure_future(fut, loop=loop)
```
Comment (cybergrind) [inline aiozk/recipes/base_watcher.py]: `ensure_future` is not in deprecation list, only function that is coroutines are affected

**Comment by cybergrind** [comment]
@decaz good work. could you please split this into two different PR's (or at least rebase to create two separate commits)? One with style changes (removing object, changing super calls to syntax without parameters) and another with loop-related changes.

File: aiozk/recipes/allocator.py [hunk 2]
```diff
@@ -72,11 +72,13 @@ def validate(self, new_allocation):
         asyncio.ensure_future(self.monitor_member_changes())
 
     async def add(self, new_item):
-        new_set = self.full_set.copy().add(new_item)
```
Comment (cybergrind) [inline aiozk/recipes/allocator.py]: This should be in a separate commit with an appropriate test for the recipe. I understand that it is not working and why, but probably after fix it still isn't working.

**Comment by decaz** [comment]
@cybergrind I removed all style changing commits. Also I've turned back loop argument to `asyncio.ensure_future` and `asyncio.open_connection`.

File: aiozk/recipes/base_watcher.py (hunk 1 above)
Comment (decaz) [reply aiozk/recipes/base_watcher.py]: Yes, and `open_connection` too. Turned them back.

**Comment by cybergrind** [comment]
@decaz uploaded as 0.18.0
